.github/workflows/build_config_server.yaml

```diff
@@ -31,21 +31,14 @@ jobs:
     - uses: actions/checkout@v4
 
     - name: Build Image
-      run: ./build.ps1 -Name ${{ env.IMAGE_NAME }} -Registry ${{ env.REGISTRY }}
+      run: ./build.ps1 -Name ${{ env.IMAGE_NAME }} -Registry ${{ env.REGISTRY }} -DirectoryName "config-server"
 
-    - name: Login to private container registry
-      if: ${{ github.event_name == 'pull_request' }}
+    - name: Login to container registry
```

Comment:
What's the consumption impact of not publishing to Docker Hub anymore? If this means that using the container requires extra command-line arguments to specify the private container registry, we should have instructions for that in the readme.

Reply:
https://github.com/SteeltoeOSS/Samples/blob/main/CommonTasks.md is the place we generally point to for Steeltoe containers, please update there as well.

Reply:
Logged an issue for followup https://github.com/SteeltoeOSS/Samples/issues/341